qiime2/core/cite.py

```diff
@@ -0,0 +1,68 @@
+import os
+import pkg_resources
+import collections
+
+import bibtexparser as bp
+
+CitationRecord = collections.namedtuple('CitationRecord', ['type', 'fields'])
+
+
+class Citations(collections.OrderedDict):
+    @classmethod
+    def load(cls, path, package=None):
+        if package is not None:
+            root = pkg_resources.resource_filename(package, '.')
+            root = os.path.abspath(root)
+            path = os.path.join(root, path)
+
+        parser = bp.bparser.BibTexParser()
+        # Downstream tooling is much easier with unicode. For actual latex
+        # users, use the modern biber backend instead of bibtex
```

Comment:
[👍 ](https://www.google.com/search?q=the+biebs&source=lnms&tbm=isch&sa=X&ved=0ahUKEwiHqP2Lka7aAhVEKawKHTeZAEUQ_AUICigB&biw=1680&bih=988&dpr=2)